server/app/import.go

```diff
@@ -58,7 +58,7 @@ func (a *App) ImportArchive(r io.Reader, opt model.ImportArchiveOptions) error {
 			return err
 		}
 
-		dir, filename := path.Split(hdr.Name)
+		dir, filename := filepath.Split(hdr.Name)
```

Comment:
@wiggin77, while this fixes the issue for me, I think there may be more to it. I checked and this code shipped in v0.15, where it is working correctly. What changed is we now generate the archive at build-time, and I *think* that causes the reader to  use OS-dependent path separators, but I'm not 100% sure.

Reply:
Switching to `filepath` is what fixed it.  `path` only handles slash, but not backslash.  `filepath` handles both.  The hardcoded paths used slash, but the generated archive used whatever the filesystem decided.  In the case of generating the archive on windows it ended up with backslashes.  `filepath` will handle either case so import won't care which OS generated the archive.